src/main/g8/src/main/scala/com/lightbend/akka/sample/HelloAkka.scala

```diff
@@ -0,0 +1,97 @@
+//#full-example
+package com.lightbend.akka.sample
+
+import akka.actor.{ ActorRef, ActorSystem, Props, Actor, ActorLogging }
+import scala.concurrent.duration._
+import scala.io.StdIn
+
+//#greeter-companion
+//#greeter-messages
+object Greeter {
+  //#greeter-messages
+  def props(message: String, printerActor: ActorRef): Props = Props(new Greeter(message, printerActor))
+  //#greeter-messages
+  case class WhoToGreet(who: String)
+  case object Greet
+}
+//#greeter-messages
+//#greeter-companion
+
+//#greeter-actor
+class Greeter(message: String, printerActor: ActorRef) extends Actor {
+  import Greeter._
+  import Printer._
+
+  var greeting = ""
+
+  def receive = {
+    case WhoToGreet(who) => greeting = s"$message, $who"
+    case Greet           =>
+      //#greeter-send-message
+      printerActor ! Greeting(greeting)
+      //#greeter-send-message
+  }
+}
+//#greeter-actor
+
+//#printer-companion
+//#printer-messages
+object Printer {
+  //#printer-messages
+  def props: Props = Props[Printer]
+  //#printer-messages
+  case class Greeting(greeting: String)
+}
+//#printer-messages
+//#printer-companion
```

Comment:
Yes, it is unfortunate. That is why I added a section in the IntelliJ docs for how to remove them with find/replace. My fear with inlining or having a separate source file is that things get out of sync. However, I am happy to update to whatever approach you prefer.